src/bun.js/javascript.zig

```diff
@@ -3616,13 +3607,24 @@ pub const VirtualMachine = struct {
                 .data => |data| {
                     IPC.log("Received IPC message from parent", .{});
                     if (this.globalThis) |global| {
+                        if (data.isObject()) {
+                            if (data.get(global, "cmd")) |prop| {
+                                if (prop.isString()) {
+                                    if (prop.toString(global).getZigString(global).hasPrefix("NODE_")) {
+                                        Process__emitInternalMessageEvent(global, data);
+                                        return;
+                                    }
+                                }
+                            }
+                        }
                         Process__emitMessageEvent(global, data);
                     }
                 },
             }
         }
 
         pub fn handleIPCClose(this: *IPCInstance) void {
+            Output.scoped(.IPC, false)("IPCInstance#handleIPCClose\n", .{});
```

Comment:
Ditto re: Output.scoped inline + scope inside of a scope. This can be a `ipc_instance("handleIPCClose", .{})` or something along those lines.